USER_GUIDE.md

```diff
@@ -297,8 +310,8 @@ services:
     name: myapp
     env: node
     plan: starter
-    buildCommand: yarn
-    startCommand: blitz start --production -H 0.0.0.0
+    buildCommand: yarn; blitz build
+    startCommand: yarn next start -p 80 -H 0.0.0.0
```

Comment:
I didn't think `-p 80` was required?

Reply:
I'm not sure if it is either, I sort of assumed it would default to port 3000 or 8000 or 8080 or some development port.  Felt like being explicit couldn't hurt.

Reply:
Ok yeah render will auto detect whatever port is being used, so we can leave this out of the guide.